open-api/rest-catalog-open-api.yaml

```
@@ -537,6 +537,108 @@ paths:
         5XX:
           $ref: '#/components/responses/ServerErrorResponse'
 
+  /v1/{prefix}/namespaces/{namespace}/tables/{table}/preplan:
+    parameters:
+      - $ref: '#/components/parameters/prefix'
+      - $ref: '#/components/parameters/namespace'
+      - $ref: '#/components/parameters/table'
+    post:
+      tags:
+        - Catalog API
+      summary: Find plan-tasks based on a plan context.
```

Maybe this? (I cannot come up with a good summary either, would like to know other suggestions)

```suggestion
      summary: Create a list of plan tasks that can be used to later when planning a table scan
```